dashboard/app/models/resource.rb

```diff
@@ -97,6 +97,12 @@ def summarize_for_lesson_edit
     }
   end
 
+  def serialize_scripts
+    if Rails.application.config.levelbuilder_mode
+      lessons.map(&:script).each(&:write_script_json)
```

Comment:
I think we might want to insert `.uniq` into this chain, so that we do not serialize extra times when the lesson appears multiple times within a single script